cluster-http/src/test/scala/akka/management/cluster/scaladsl/ClusterMembershipCheckSpec.scala

```diff
@@ -25,6 +25,7 @@ class ClusterMembershipCheckSpec
         new ClusterMembershipCheckSettings(Set(MemberStatus.Up)))
 
       chc().futureValue shouldEqual false
+      system.terminate()
```

Comment:
shouldn't this be in `afterAll`, and use `shutdown()` (in extended TestKit)?

Reply:
Thanks @patriknw.


Surprisingly, a single `terminate` on only one of the tests fixed the issue but I agree that using `afterAll+shutdown` is the correct solution.